internal/debug/flags.go

```diff
@@ -86,13 +86,40 @@ var (
 		Name:  "trace",
 		Usage: "Write execution trace to the given file",
 	}
+	splunkurlFlag = cli.StringFlag{
```

Comment:
Please add prefix `log.` to all those flag names, e.g. "log.splunkurl".

Reply:
Done.